src/subcommand/wallet.rs

```diff
@@ -33,6 +36,7 @@ pub(crate) enum Wallet {
 impl Wallet {
   pub(crate) fn run(self, options: Options) -> Result {
     match self {
+      Self::Balance(balance) => balance.run(options),
```

Comment:
We don't actually need a struct here, because balance doesn't take any options. This can be:
```suggestion
      Self::Balance => balance::run(options),
```

Reply:
cool! Still figuring out the internal structure here. thats helpful